docs-js/orchestration/chat-completion.mdx

```diff
@@ -136,6 +136,57 @@ const response = await orchestrationClient.chatCompletion({
 });
 ```
 
+### Remote Prompt Registry
+
+You can define template content within the templating configuration, or refer to a template defined in the [Prompt Registry](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/prompt-registry).
+
+```ts
+const orchestrationClient = new OrchestrationClient({
+  llm,
+  templating: {
+    template_ref: {
+      name: 'my-get-capital-template',
+      scenario: 'my-scenario',
+      version: '0.0.1'
+    }
+  }
+});
+
+const response = orchestrationClient.chatCompletion({
+  inputParams: { country: 'France' }
+});
+```
+
+A prompt template can be referenced either by ID, or by a combination of name, scenario and version.
+For details on storing a template in the Prompt Registry, refer to [this guide](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/create-prompt-template-imperative).
+
+### Local Prompt Template
+
+You can define the template content by passing the contents of a YAML file as a string in the `templating` configuration.
+This is useful for testing a prompt template locally before storing it in the [Prompt Registry](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/prompt-registry).
+
+```ts
+import { readFileSync } from 'fs';
+import { OrchestrationClient } from '@sap-ai-sdk/orchestration';
+
+// Read the YAML file containing the prompt template
+const yamlTemplate = readFileSync('./path/to/prompt-template.yaml', 'utf-8');
+
+const orchestrationClient = new OrchestrationClient({
+  llm,
+  templating: yamlTemplate
+});
+
+const response = orchestrationClient.chatCompletion({
+  inputParams: { country: 'France' }
+});
+```
+
+The YAML string is parsed and validated against the expected schema.
+If the YAML is invalid or does not conform to the schema, an error will be thrown.
+
+This [YAML file](https://github.com/SAP/ai-sdk-js/blob/main/test-util/data/orchestration/orchestration-chat-completion-yaml-template.yaml) illustrates the format in which a prompt template can be defined for use in the templating configuration.
```

Comment:
[q] I wonder if we should link the schema here so that user knows how to construct such YAML files? `PromptTemplatePostRequestSchema`

Reply:
Good idea, I think we do link to some official help.sap docs here. I'll check and add if its missing